javaslang/src/test/java/javaslang/collection/QuickSort.java

```diff
@@ -0,0 +1,33 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _______
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  //  /\__\   JΛVΛSLΛNG
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/ \ /__\ \   Copyright 2014-2017 Javaslang, http://javaslang.io
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/\__\_/  \_//  \__/\_____/   Licensed under the Apache License, Version 2.0
+ */
+package javaslang.collection;
+
+import javaslang.Function1;
+import org.junit.Test;
+
+import static java.lang.Integer.signum;
+import static javaslang.API.List;
+
+public class QuickSort {
```

Comment:
I think this test will never run. By convention maven-surefire-plugin look for classes that match the following patterns:

* \*\*/Test*.java
* \*\*/*Test.java
* \*\*/*TestCase.java

(source: http://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html)